examples/opmsave_inspect.cpp

```diff
@@ -0,0 +1,78 @@
+/*
+  Copyright 2020 Equinor.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <config.h>
+
+#include <ebos/hdf5serializer.hh>
+
+#include <opm/simulators/timestepping/SimulatorTimer.hpp>
+
+#include <boost/date_time.hpp>
+
+#include <fmt/format.h>
+
+#include <array>
+#include <iostream>
+#include <string>
+
+int main(int argc, char** argv)
+{
+    if (argc < 2) {
+        std::cerr << "Need one parameter, the .SAVE file to inspect\n";
+        return 1;
+    }
+
+    Opm::HDF5Serializer ser(argv[1], Opm::HDF5File::OpenMode::READ);
+
+    std::tuple<std::array<std::string,5>,int> header;
+    try {
+        ser.read(header, "/", "simulator_info");
+    } catch(...) {
+        std::cerr << "Error reading data from file, is it really a .SAVE file?\n";
+        return 2;
+    }
+
+    const auto& [strings, procs] = header;
+    std::cout << "Info for " << argv[1] <<":\n";
+    std::cout << fmt::format("\tSimulator name: {}\n"
+                             "\tSimulator version: {}\n"
+                             "\tCompile time stamp: {}\n"
+                             "\tCase name: {}\n"
+                             "\tNumber of processes: {}\n",
+                             strings[0], strings[1], strings[2], strings[3], procs);
+
+    std::cout << fmt::format("\tLast report step: {}\n", ser.lastReportStep());
+    const std::vector<int> reportSteps = ser.reportSteps();
+    for (int step : reportSteps) {
+        Opm::SimulatorTimer timer;
+        try {
+            ser.read(timer, "/report_step/" + std::to_string(step),
+                     "simulator_timer");
```

Comment:
You have `format.h` so you don't need `to_string()` here:
```C++
ser.read(timer, fmt::format("/report_step/{}", step), "simulation_timer");
```

Reply:
yeh, first iteration had no fmt..